src/test/java/de/drachir000/utils/config/SimpleConfigLibTest.java

```diff
@@ -49,4 +52,24 @@ public void testConfigurationFromString() {
 		
 	}
 	
+	@Test
+	public void testFiles() throws IOException {
+		
+		Configuration configuration = SimpleConfigLib.emptyConfiguration();
+		File file = new File("settings.json");
+		
+		if (file.exists())
+			assertTrue(file.delete());
+		
+		configuration.setString("key", "value");
+		
+		SimpleConfigLib.save(configuration, file);
+		
+		assertTrue(file.exists());
+		
+		Configuration loadedConfig = SimpleConfigLib.load(file);
```

Comment:
As already mentioned. What happens, when the File doesn't exist or is bank? I belive it would be best to create an empty Configuration object then.

Reply:
Implemented in https://github.com/Drachir000/SimpleConfigLib/commit/42a8f41e4f08a57fe93b688710434b6ab987648b